.github/issue_template.md

```diff
@@ -0,0 +1,14 @@
+**Description:**
```

Comment:
This assumes that the issue being filed is a bug, issues can be filed for other reasons like feature requests, and usage questions.

Reply:
This is a good template for a bug though.

Reply:
I've made the template generic to encapsulate other reasons for an issue submission.